h/templates/layouts/base.html

```diff
@@ -10,7 +10,13 @@
       {% endfor -%}
     {% endblock %}
 
-    <title>{% block title %}Hypothesis{% endblock %}</title>
+    <title>
+      {%- block title -%}
+        {%- block page_title %}{% endblock -%}
+        {%- if self.page_title() %} | {% endif -%}
```

Comment:
The block is not always defined it seems because it's not an inheritance / overriding of the title block, it's a separate, optional block that the child may define.

I've just tried it. I'm pushing this change.